nox/virtualenv.py

```diff
@@ -182,6 +196,92 @@ def _resolved_interpreter(self):
         self._resolved = InterpreterNotFound(self.interpreter)
         raise self._resolved
 
+    @property
+    def _runtime_interpreter(self):
```

Comment:
This property above seems to be mostly duplicated effort for locating the interpreter that is done elsewhere

Reply:
This is my attempt to follow `_resolved_interpreter`, what do you suggest?